ui/pages/onboarding-flow/welcome/index.scss

```diff
@@ -1,4 +1,9 @@
-@import "./react-gallery/carousel.min.css";
+@use "design-system";
+// by using @use, and dropping the file extension, sass will inline the
+// stylesheet instead of creating generating an `import(carousel.css)`. This is
+// desirable because it reduces the number of render-blocking requests on app
+// startup.
```

Comment:
This is great! Does this happen for the @metamask/design-tokens stylesheet as well?

Reply:
Comment is a little misleading; `@use` will always inline (AFAIK), even with the file extension. `@import` will inline if there is no file extension. Confusing? Yes. Since `@import` is deprecated I figured we might as well switch.

Which reminds me... that the build process doesn't need to copy the style sheet to the output dir anymore. I'll create a follow up PR to remove that step from the build process, and fix up this comment in that PR as well. Self-assigned in this issue: https://github.com/MetaMask/metamask-extension/issues/23094

Reply:
FYI: I've opened https://github.com/MetaMask/metamask-extension/pull/23278 to finally resolve this comment and [the corresponding issue](https://github.com/MetaMask/metamask-extension/issues/23094))